src/Classes/PassiveSpecListControl.lua

```diff
@@ -5,44 +5,132 @@
-		self:RenameSpec(newSpec, "Copy Tree", true)
+	self.showRowSeparators = true
+
+	-- Copy button is in the middle, the others spread out from there
+	self.controls.copy = new("ButtonControl", {"TOPLEFT", self, "TOPLEFT"}, (width -60 ) / 2, -25, 60, 18, "Copy", function()
+		-- If there is a selection add new spec after it
+		self:OnSelCopy(self.selIndex, self.selValue, self.selections[1])
 	end)
 	self.controls.copy.enabled = function()
-		return self.selValue ~= nil
+		return self.selValue ~= nil and #self.selections == 1
 	end
 	self.controls.delete = new("ButtonControl", {"LEFT",self.controls.copy,"RIGHT"}, 4, 0, 60, 18, "Delete", function()
-		self:OnSelDelete(self.selIndex, self.selValue)
+		self:OnSelDelete()
 	end)
+	self.controls.delete.tooltipText = "Use Ctrl-Left Click to multi select trees below"
 	self.controls.delete.enabled = function()
 		return self.selValue ~= nil and #self.list > 1
 	end
-	self.controls.rename = new("ButtonControl", {"BOTTOMRIGHT",self,"TOP"}, -2, -4, 60, 18, "Rename", function()
+	self.controls.convert = new("ButtonControl", {"LEFT",self.controls.delete,"RIGHT"}, 4, 0, 60, 18, "Convert", function()
+		local text = ""
+		-- Convert one or more specs (Trees)
+		local plural = #self.selections == 1 and "" or "s"
+		if #self.selections > 0 then
+			-- validate entries are not the latest version, run backwards so we can remove entries safely
+			local totalSelections = #self.selections
+			for selId = totalSelections, 1, -1 do
+				local spec = self.list[self.selections[selId]]
+				if spec.treeVersion == latestTreeVersion then
+					t_remove(self.selections, self.selections[selId])
+				end
+			end
+			-- Create controls for the popup
+			local controls = { }
+			local numMsgLines = 2
+			controls.label1 = new("LabelControl", nil, 0, 20, 0, 16, "Are you sure you want to convert the following tree"..plural.."?")
+			-- Collect spec names, sorted in the order shown on screen and show them to the user for confirmation. 
+			t_sort(self.selections)
+			for selId = 1, #self.selections do
+				local spec = self.list[self.selections[selId]]
+				-- collect the spec names so we can find the widest later
+				text = text..self:GetRowValue(1, index, spec).."\n"
+				t_insert(controls, new("LabelControl", {"LEFT",nil,"TOPLEFT"}, 10, 20 + numMsgLines * 18, 0, 16, self:GetRowValue(1, index, spec)..'^7'))
+				numMsgLines = numMsgLines + 1
+			end
+			numMsgLines = numMsgLines + 1
+			controls.label2 = new("LabelControl", {"LEFT",nil,"TOPLEFT"}, 10, 20 + numMsgLines * 18, 0, 16, "Note that some or all of the passives may have been de-allocated due to changes in the tree.")
+			controls.label3 = new("LabelControl", {"LEFT",controls.label2,"BOTTOMLEFT"}, 0, 10, 0, 16, "You can switch back to the old tree using the tree selector at the bottom left.")
+			controls.label4 = new("LabelControl", {"LEFT",controls.label3, "BOTTOMLEFT"}, 5, 15, 0, 16, "Where do you want the new tree"..plural.." placed?")
```

Comment:
```suggestion
			controls.label3 = new("LabelControl", {"LEFT",controls.label2,"BOTTOMLEFT"}, 0, 10, 0, 16, "You can switch back to the old tree".. plural .. " using the tree selector at the bottom left.")
			controls.label4 = new("LabelControl", {"LEFT",controls.label3, "BOTTOMLEFT"}, 0, 15, 0, 16, "New tree placement:")
```

Reworded this prompt because having a second question mark in the dialog window makes me double check which question "Yes" and "No" is answering.